File: doc/_i18n/en/tutorials/introduction/dual-arm-controller/p2.py
# In the run callback
def run_callback(self):
    # ...
    elif (
            self._phase == ControllerPhase.STARTED
            and self.postureTask.eval().norm() < 0.01
            and self.postureTask.speed().norm() < 0.01
        ):
            self._phase = ControllerPhase.MOVE
            self.qpsolver.addTask(self._urEndEffectorTask)
    elif self._phase == ControllerPhase.STARTED:
        self._urEndEffectorTask.reset()
    # ...
    return True

File: doc/_i18n/jp/tutorials/introduction/dual-arm-controller/p2.cpp
// run関数内
bool DualArmController::run() {
  // ...
  else if (phase_ == STARTED && postureTask->eval().norm() < 0.01 &&
      postureTask->speed().norm() < 0.01) {
    phase_ = MOVE;
    urEndEffectorTask_->selectActiveJoints(solver(), urJoints_);
  } else if (phase_ == STARTED) {
    urEndEffectorTask_->reset();
  }
  // ...
  return mc_control::MCController::run();
}